cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -47,8 +74,14 @@ Cost::
 This planner uses the statistics service in Neo4j to assign cost to alternative plans and picks the cheapest one.
 While this should lead to superior execution plans in most cases, it is still under development.
 
-By default, Neo4j {neo4j-version} will use the cost planner for all queries.
-You can force it to use a specific planner by using the `cypher.planner` configuration setting (see <<operations-manual#config_cypher.planner, Operations Manual -> Configuration Settings>>), or by prepending your query with `CYPHER planner=cost` or `CYPHER planner=rule`.
+[options="header"]
+|===
+| Preparser option | Description | Default?
+| `planner=rule` | This will force the query to use the rule planner.  |
+| `planner=cost` | This will force the query to use the cost planner. As this is the default planner, it is not necessary to use this option explicitly. | X
+|===
+
+It is also possible to force queries to use a specific planner by using the `cypher.planner` configuration setting (see <<operations-manual#config_cypher.planner, Operations Manual -> Configuration Settings>>).
```

Comment:
Is this really a 'force' option? According to the inline docs in GraphDatabaseSettings, this just sets the default planner. We should re-word this accordingly.

Reply:
👍